lib/utils/index.js

```diff
@@ -1058,17 +1058,31 @@ function unsigned_url_prefix(
   return prefix;
 }
 
-function base_api_url(path = [], options = {}) {
-  let cloudinary = ensureOption(options, "upload_prefix", UPLOAD_PREFIX);
-  let cloud_name = ensureOption(options, "cloud_name");
-  let encode_path = unencoded_path => encodeURIComponent(unencoded_path).replace("'", '%27');
-  let encoded_path = Array.isArray(path) ? path.map(encode_path) : encode_path(path);
-  return [cloudinary, "v1_1", cloud_name].concat(encoded_path).join("/");
+function base_api_url_v1() {
+  return base_api_url('v1_1');
+}
```

Comment:
Maybe rename to `base_api_url_v1_1`?

Reply:
Same thought I had, if we have a specific version here and we put the version in the name then I guess it makes sense to be specific in the name too.